services/src/main/java/org/exoplatform/news/storage/jcr/JcrNewsStorage.java

```diff
@@ -634,6 +609,8 @@ public News updateNews(News news, String updater) throws Exception {
       newsNode.setProperty("exo:body", news.getBody());
       newsNode.setProperty("exo:dateModified", Calendar.getInstance());
       newsNode.setProperty(EXO_NEWS_LAST_MODIFIER, updater);
+      newsNode.setProperty("exo:pinned", news.isPublished());
```

Comment:
Normally this method is called even when we update the simple fields (title, summary, content) without updading publish, so IMO we should not set the property exo:pinned in that case which may causes a problem (to be verified)
I think that we should keep publishNews and unpublishNews methods in jcr storage layer with only setting exo:pinned property (without creating symlink)
cc @ahamdi